src/elements/content-sidebar/SidebarNav.js

```diff
@@ -108,15 +102,9 @@ const SidebarNav = ({
                     )}
                 </SidebarNavTablist>
 
-                {hasBoxSign && onBoxSignClick && (
+                {hasBoxSign && onBoxClickRequestSignature && (
                     <div className="bcs-SidebarNav-secondary">
-                        <SidebarNavSign
-                            blockedReason={boxSignBlockedReason}
-                            data-resin-target={SIDEBAR_NAV_TARGETS.SIGN}
-                            onClick={() => onBoxSignClick({ fileId })}
-                            status={boxSignStatus}
-                            targetingApi={boxSignTargetingApi}
-                        />
+                        <SidebarNavSign data-resin-target={SIDEBAR_NAV_TARGETS.SIGN} data-testid="sidebarsign" />
```

Comment:
should we use `data-resin-target` and `data-testid` directly in `SidebarNavSignButton`? this way we won't need to pass the props from here to `SidebarNavSign` and then to `SidebarNavSignButton`